templates/sensu-redis.json.j2

```diff
@@ -1,6 +1,14 @@
 {
   "redis": {
+{% if redis_password %}
+    "password": "{{ redis_password }}",
+{% endif %}
+{% if redis_sentinels %}
+    "sentinels": {{ redis_sentinels | to_nice_json(indent=4) }},
```

Comment:
I didn't know it was exclusive to Ansible 2.2, my bad. It is indeed not necessary.